gopro2gpx/ffmpegtools.py

```diff
@@ -9,11 +9,44 @@
 import subprocess
 import re
 import json
+import platform
+from collections import namedtuple
+
+Version = namedtuple('Version', ['major', 'medium', 'minor'])
+
+def default_fftools():
+    if platform.system() == 'Windows':
+        ffmpeg, ffprobe = "ffmpeg.exe", "ffprobe.exe"
+    else:
+        ffmpeg, ffprobe = "ffmpeg", "ffprobe"
+
+    return ffmpeg, ffprobe
+
 
 class FFMpegTools:
 
-    def __init__(self, config):
-        self.config = config
+    def __init__(self, ffprobe=None, ffmpeg=None):
+        default_ffmpeg, default_ffprobe = default_fftools()
+
+        self.ffmpeg = ffmpeg if ffmpeg else default_ffmpeg
+        self.ffprobe = ffprobe if ffprobe else default_ffprobe
+
+        self.version = self.getVersion()
+
+        self.use_json_format = False
+        if self.version.major >= 4:
```

Comment:
Now using the `major` field of the semver comparison.

Reply:
roger